cookiecutter/repository.py

```diff
@@ -23,6 +24,11 @@ def is_repo_url(value):
     return bool(REPO_REGEX.match(value))
 
 
+def is_zip_file(value):
+    """Return True if value is a zip file."""
+    return value.endswith('.zip')
```

Comment:
AFAIK file extensions on Windows are not case sensitive, so maybe we want to change this ``return value.lower().endswith('.zip')``.